modules/web/src/app/node-data/basic/provider/gcp/component.ts

```diff
@@ -266,7 +262,6 @@ export class GCPBasicNodeDataComponent extends BaseFormValidator implements OnIn
     this.diskTypes = [];
     this.selectedDiskType = '';
     this.diskTypeLabel = DiskTypeState.Empty;
-    this._diskTypesCombobox.reset();
```

Comment:
This method indirectly triggers the `onDiskTypeChange` method so after removing this, it might not clear the `this._nodeDataService.nodeData.spec.cloud.gcp.diskType` value when you change the zone. Can you please verify that it still works?

Reply:
yeh i test it and it's working as expected, i will update some changes for the edit dialog
i don't see a reason to clear the value, we just need to update it with the default value, and the `_setDefaultDiskType` and `_setDefaultMachineType` are trigered when zone changes and the `onDiskTypeChange` and `onMachineTypeChange` method update the  `this._nodeDataService.nodeData.spec.cloud.gcp.diskType`